app/lib/code_sets/trs.rb

```diff
@@ -53,7 +53,7 @@ module Trs
       "withdrawn" => "Withdrawn",
       "deferred" => "Deferred",
       "awarded" => "Awarded",
-      "recommended_for_award" => "Approved",
+      "recommended_for_award" => "Awarded",
```

Comment:
https://hedgedoc.teacherservices.cloud/4_LiHuZXRzO6GwuVS12dkw?view#Statuses corrected to accepted statuses